Makefile

```diff
@@ -1,7 +1,8 @@
 blank:
-	@echo "Downloading ona core dependencies and running migrations for you (hopefully into a virtualenv)... \n (This will take a while, go grab a coffee or something.)\n\n"
-	python2 -m pip install -r requirements/makefile_dependencies.pip
-	python2 -m pip install -e .
-	python2 manage.py syncdb --noinput
-	python2 manage.py migrate
-	@echo "Start the app with \`python2 manage.py runserver --nothreading\`"
+  @echo "Downloading ona core dependencies and running migrations for you (hopefully into a virtualenv)... \n (This will take a while, go grab a coffee or something.)\n\n"
+  python2 -m pip install -r requirements/base.pip
+  python2 -m pip install -r requirements/dev.pip
+  # what does syncdb do?
```

Comment:
looks like nothing anymore, https://stackoverflow.com/questions/29683494/what-should-i-use-instead-of-syncdb-in-django-1-9. I think it's fine to remove these 2 lines